static/src/javascripts/components/raven-js/raven.js

```diff
@@ -1410,7 +1410,14 @@ var Raven = {
      */
     lastEventId: function() {
         return lastEventId;
-    }
+    },
+
+    /*
+     * Has raven been configured?
+     *
+     * @return {boolean}
+     */
+    isSetup: isSetup
```

Comment:
Good shout. I'll experiment with the latest version (3.8.0) and if it's compatible I'll upgrade it. We're still using version 1.1.6